auth/src/main/java/com/firebase/ui/auth/ui/ProgressDialogHolder.java

```diff
@@ -18,7 +18,6 @@ public ProgressDialogHolder(Context context) {
 
     private void showLoadingDialog(String message) {
```

Comment:
So we really can't kill this class yet? 😭

Reply:
Right now it just acts as a "fallback" implementation.  If you were to add a new activity or fragment and not override the progress methods to do something special, it would use a dialog.

Once phone auth is deleted we can remove the base implementation, I think.